client/pages/result.jsx

```diff
@@ -1,21 +1,48 @@
 import React from 'react';
 import Accordion from '../components/accordion';
+import AppDrawer from '../components/app-drawer';
 
 export default class Result extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      view: 'results'
+      view: 'results',
+      currentPlanId: parseInt(window.location.hash.substring(15)),
```

Comment:
What is the significance of `15` here? Please message me about it. In general we don't want "magic numbers" that don't really mean anything by themselves.